sdk/translation/Azure.AI.Translation.Text/tsp-location.yaml

```diff
@@ -0,0 +1,3 @@
+directory: specification/translation/text
+commit: 0f0de365a7125c3d1085ad94677fda264a70c977
+repo: MikeyMCZ/azure-rest-api-specs
```

Comment:
Is the plan to consume the public repo at a later time. Please have a bug so you don't forget.

Reply:
We definitely do not want to merge in this state

Reply:
We have PR for cadl repository: https://github.com/Azure/azure-rest-api-specs/pull/23231 Once it will be merged, this will be updated.